free_fleet_adapter/free_fleet_adapter/nav1_robot_adapter.py

```diff
@@ -314,6 +317,8 @@ def __init__(
         self.tf_buffer = tf_buffer
 
         self.map_name = self.robot_config_yaml['initial_map']
+        self.frame_map = self.robot_config_yaml['map_frame']
+        self.frame_robot = self.robot_config_yaml['robot_frame']
```

Comment:
Check if `self.robot_config_yaml` does indeed contain `map_frame` or `robot_frame`, otherwise use default values. This will help to prevent breaking existing behavior for other configs